e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/fee-record-correction/feature-flag-enabled/record-correction-log-correct-records.spec.js

```diff
@@ -41,13 +43,13 @@ const correctionEntity1 = FeeRecordCorrectionEntityMockBuilder.forFeeRecordAndIs
   .withPreviousValues({
     facilityUtilisation: 600000,
     feesPaidToUkefForThePeriod: 12345,
-    feesPaidToUkefForThePeriodCurrency: 'GBP',
-    facilityId: '123456',
+    feesPaidToUkefForThePeriodCurrency: CURRENCY.GBP,
+    facilityId: firstFeeRecord.facilityId,
```

Comment:
The current facility id on the fee record should match the corrected value not the previous value please

Reply:
done